pkg/daemon/ceph/osd/volume.go

```diff
@@ -200,25 +268,25 @@ func sanitizeOSDsPerDevice(count int) string {
 }
 
 func getCephVolumeSupported(context *clusterd.Context) (bool, error) {
-	_, err := context.Executor.ExecuteCommandWithOutput(false, "", cephVolumeCmd, "lvm", "batch", "--prepare")
-	if err != nil {
-		if cmdErr, ok := err.(*exec.CommandError); ok {
-			exitStatus := cmdErr.ExitStatus()
-			if exitStatus == int(syscall.ENOENT) || exitStatus == int(syscall.EPERM) {
-				logger.Infof("supported version of ceph-volume not available")
-				return false, nil
-			}
-			return false, fmt.Errorf("unknown return code from ceph-volume when checking for compatibility: %d", exitStatus)
-		}
-		return false, fmt.Errorf("unknown ceph-volume failure. %+v", err)
-	}
+	// _, err := context.Executor.ExecuteCommandWithOutput(false, "", cephVolumeCmd, "lvm", "prepare")
```

Comment:
I'm removing this method in #3450. If this is blocking you, how about if you skip running it if the pvcBacked flag is set?

Reply:
This command is not blocking us, I don't think it's affecting anything at all.